src/Microsoft.AspNetCore.DataProtection/DataProtectionConfiguration.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.AspNetCore.DataProtection
     /// <para>
     /// If the developer changes the at-rest key protection mechanism, it is intended that
     /// he also change the key storage location, and vice versa. For instance, a call to
-    /// <see cref="ProtectKeysWithCertificate(string)"/> should generally be accompanied by
+    /// <c>ProtectKeysWithCertificate(string)</c> should generally be accompanied by
```

Comment:
Well, we can't ifdef a specific part of the comment. The only way is to ifdef the entire `<summary>` and we'll have two summary with just one line difference.